fe/fe-core/src/main/java/com/starrocks/lake/StarMgrMetaSyncer.java

```diff
@@ -95,6 +95,19 @@ private List<Long> getAllPartitionShardGroupId() {
 
     public static void dropTabletAndDeleteShard(List<Long> shardIds, StarOSAgent starOSAgent) {
         Preconditions.checkNotNull(starOSAgent);
+        // a short-cut in case actual data deletion is too slow
+        if (Config.meta_sync_force_delete_shard_meta) {
+            try {
+                if (!shardIds.isEmpty()) {
+                    Set<Long> shards = new HashSet<Long>(shardIds);
+                    starOSAgent.deleteShards(shards);
```

Comment:
is it possible to log the shard storage path, so the reminder data on remote storage path can be collected from logfile and do some post-action offline later.